tests/console/validate_ext4_fs.pm

```diff
@@ -0,0 +1,74 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate ext4 filesystem and system partitioning.
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+
+    select_console "root-console";
+
+    my $test_data  = get_test_suite_data();
+    my @partitions = @{$test_data->{partitions}};
+
+    foreach (@partitions) {
+
+        if ($_->{mnt_point} eq '[SWAP]') {
+
+            # Validate that the partition is created.
+
+
+            record_info("Check $_->{mnt_point}", "Verify the '$_->{mnt_point}' partition was created.");
+            assert_script_run("lsblk -n | grep SWAP", fail_message => "Partition SWAP was not created.");
+            my $fstype = script_output("cat /etc/fstab | grep swap | awk '{print \$3}'");
+            assert_equals($_->{fs_type}, $fstype, "File system on '$_->{mnt_point}' partition does not correspond to the expected one.");
+
+        }
+        else {
+
+            # Validate that the partition is created.
+
+            record_info("Check $_->{mnt_point}", "Verify the '$_->{mnt_point}' partition was created.");
+            assert_script_run("lsblk -n | grep $_->{mnt_point}\$", fail_message => "Partition $_->{mnt_point} was not created.");
+
+            # Validate that the filesystem type is the expected one.
+
```

Comment:
As I mentioned before you are adding blank lines between comments and code. Check the repo, but I think the style used is different and makes more sense to link together both. I'd only use to separate logic blocks for clarity, but after `else` or `foreach` is not needed.

Reply:
Done